include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.cpp

```diff
@@ -610,6 +610,13 @@ TimeIntBDF<dim, Number>::postprocessing() const
     operator_base->move_grid_and_update_dependent_data_structures(this->get_time());
   }
 
+  // We need to distribute the dofs for HDIV before computing the error since
+  // dealii::VectorTools::integrate_difference() does not take the periodic boundary
+  // constraints into account like MatrixFree does, hence reading the wrong value.
+  // distribute() updates the otherwise unused values.
```

Comment:
Nice comment! 

I do not understand "otherwise unused" in the last line. -> "distribute() updates those values. Note that MatrixFree does this automatically."?